packages/react/src/Card/Card.features.stories.tsx

```diff
@@ -31,19 +31,67 @@ export const Label: ComponentStory<typeof Card> = () => {
     </Card>
   )
 }
+
+export const Icon: ComponentStory<typeof Card> = () => {
```

Comment:
For visual regression testing purposes, can we add a story with _just_ the icons on their own into Storybook? We'll then have a snapshot of all the color variations in one story.